tests/integration/test_akafka_testutils.py

```diff
@@ -136,15 +136,17 @@ async def test_clear_topics_specific(kafka: KafkaFixture):
 
     # make sure messages are consumed again
     records = []
-    while True:
-        prefetched = await consumer.getmany(timeout_ms=5000)
-        if not prefetched:
+    while (fetches := 0) < 10:
```

Comment:
`for _ in range(10):` is more idiomatic.

Reply:
I changed it in [b61524f](https://github.com/ghga-de/hexkit/pull/112/commits/b61524f6ee2630c742a9e0bca67352005e6a0a60)